app/api/__init__.py

```diff
@@ -25,8 +25,8 @@
 
 # users
 api.route(UserList, 'user_list', '/users')
-api.route(UserDetail, 'user_detail', '/users/<int:id>',
-          '/notifications/<int:notification_id>/user', '/event-invoices/<int:event_invoice_id>/user')
+api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user',
+          '/event-invoices/<int:event_invoice_id>/user')
```

Comment:
No. event-invoices is correct I guess. Give a relation b/w event invoice and a user.